examples/with-redux-toolkit/package.json

```diff
@@ -1,19 +1,18 @@
 {
-  "name": "with-redux",
+  "name": "with-redux-toolkit",
   "version": "1.0.0",
   "scripts": {
     "dev": "next",
     "build": "next build",
     "start": "next start"
   },
   "dependencies": {
-    "@reduxjs/toolkit": "^1.2.5",
+    "@reduxjs/toolkit": "^1.3.6",
     "next": "latest",
-    "react": "^16.9.0",
-    "react-dom": "^16.9.0",
-    "react-redux": "^7.1.0",
-    "redux": "^3.6.0",
-    "redux-devtools-extension": "^2.13.2"
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-redux": "^7.2.0",
+    "redux": "^4.0.5"
```

Comment:
I don't think we need `redux` listed as dependency here? `redux-toolkit` should includes it?

Reply:
Correct, RTK already has Redux as a specific dependency.

Reply:
Fixed at 9bee26a